projects/igniteui-angular/src/lib/services/overlay/scroll/absolute-scroll-strategy.ts

```diff
@@ -10,29 +11,33 @@ export class AbsoluteScrollStrategy extends ScrollStrategy {
     private _overlayService: IgxOverlayService;
     private _id: string;
     private _scrollContainer: HTMLElement;
+    private _zone: NgZone;
 
     constructor(scrollContainer?: HTMLElement) {
         super(scrollContainer);
         this._scrollContainer = scrollContainer;
     }
 
     /** @inheritdoc */
-    public initialize(document: Document, overlayService: IgxOverlayService, id: string) {
+    public initialize(document: Document, overlayService: IgxOverlayService, id: string, zone?: NgZone) {
```

Comment:
@wnvko Can the zone be accessed from the service itself (could even be `@internal`) to avoid adding one more parameter?

Reply:
much better in this way.
Thank you :)